test/e2e/basic_test.go

```diff
@@ -1079,6 +1079,16 @@ var _ = Describe("Third party objects", func() {
 			}
 			return nil
 		}, 30*time.Second, 1*time.Second).Should(BeNil())
+
+		ipnodes, err := ext.IPNodes().List(api.ListOptions{})
+		Expect(err).NotTo(HaveOccurred())
+		for _, node := range ipnodes.Items {
+			node.Revision = 10000000
+			newNode, err := ext.IPNodes().Update(node)
+			Expect(err).NotTo(HaveOccurred())
+		}
+		_, err = ext.IPNodes().List(api.ListOptions{})
```

Comment:
i think that on successful update object will un-marshaled from returned data, so it is probably redundant